src/App.css

```diff
@@ -28,6 +28,38 @@
   color: #61dafb;
 }
 
+.container {
+  display: flex;
+  height: 48px;
+  margin-bottom: 16px;
+  text-align: center;
+}
+
+.id-container, .task-name-container { 
+  align-items: center;
+  justify-content: center;
```

Comment:
I think you forgot the `display: flex` because the `align-items` and `justify-content` won't work without that